lib/mbta_metro/components/accordion.ex

```diff
@@ -51,4 +58,22 @@ defmodule MbtaMetro.Components.Accordion do
     <%= render_slot(@extra) %>
     """
   end
+
+  def accordion(assigns) do
+    ~H"""
+    <details id={@id} class="group w-full bg-white" open={@open}>
+      <summary class="text-slate-500 p-2 flex cursor-pointer list-none gap-2 p-2 relative">
+        <%= render_slot(@heading) %>
+        <.icon
+          name="chevron-up"
+          class="group-open:rotate-180 w-4 h-4 absolute top-3 right-3 fill-blue-500"
+        />
+      </summary>
+      <div class=" border-t border-solid border-slate-300 p-2">
+        <%= render_slot(@content) %>
+      </div>
+    </details>
+    <%= render_slot(@extra) %>
```

Comment:
Might not need this `@extra`, I'd only added it to the accordion as a crude way to show an error message outside containment. 🤷🏼‍♀️